harmonyfrontend/src/components/library/movies-list.jsx

```diff
@@ -4,6 +4,7 @@ import { observer } from "mobx-react";
 import Loading from "../../components/loading";
 import Movie from "./movie";
 import "./library.scss";
+import sortableTitle from "./titlesort";
```

Comment:
@dylanpizzo same comment as below, not a fan of a file in the /components/ directory which exports a function that isn't a component. Should instead be a utility method added to the model.